src/BusinessLayer/TelemetryReporting.h

```diff
@@ -31,8 +42,13 @@ class TelemetryReporting
 
    void writeFloatIntoData(unsigned char* data, int index, const float& value);
    void sendData(const unsigned char* data, int length);
+   void connectToView();
+   void sendCmu(unsigned char cmuDataIndex);
+   void sendMppt(unsigned char mpptDataIndex);
 
 private:
    QIODevice& outputDevice_;
    VehicleData& vehicleData_;
+   QSerialPort& serialPort_;

```

Comment:
The reason I added my own temporary(makeshift solution) serialPort is because it requires specifically a serialPort in one of the methods. Will remove later when Peripheral interface is made.